homeassistant/components/zha/core/helpers.py

```diff
@@ -170,6 +170,7 @@ def async_get_zha_device(hass: HomeAssistant, device_id: str) -> ZHADevice:
     device_registry = dr.async_get(hass)
     registry_device = device_registry.async_get(device_id)
     zha_gateway: ZHAGateway = hass.data[DATA_ZHA][DATA_ZHA_GATEWAY]
+    assert registry_device
```

Comment:
`registry_device.identifiers` would have failed if `registry_device` was None

Reply:
I'd rather raise ValueError explicitly if the device_id is missing than raising AttributeError implicitly.